File: app/src/main/java/com/blog/demo/component/fragment/LifeFragmentActivity.java
package com.blog.demo.component.fragment;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.fragment.app.FragmentActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import com.blog.demo.R;

public class LifeFragmentActivity extends FragmentActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_component_life_fragment);

        if (findViewById(R.id.fragment_right) != null) {
            FragmentManager fm = getSupportFragmentManager();
            FragmentTransaction ft = fm.beginTransaction();
            ft.add(R.id.fragment_right, new FragmentRight());
            ft.commitAllowingStateLoss();
        }
    }

}

File: app/src/main/java/com/blog/demo/component/fragment/LifeFragmentActivity.kt
package com.blog.demo.component.fragment

import android.os.Bundle
import android.view.View
import androidx.fragment.app.FragmentActivity
import com.blog.demo.R

class LifeFragmentActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_component_life_fragment)

        var fragmentView = findViewById<View?>(R.id.fragment_right)
        if (fragmentView != null) {
            val fm = supportFragmentManager
            val ft = fm.beginTransaction()
            ft.add(R.id.fragment_right, FragmentRight())
            ft.commitAllowingStateLoss()
        }
    }

}